Add user import schemas and fix some issues

matepager opened the pull request:
- Added schemas and examples for import files that contain only user data.
- The `activation_date` field is now optional in all of the schemas, as described in the documentation.
- Fixed some minor file formatting issues.

matepager committed:
Add user import schemas and fix some issues

- Added schemas and examples for import files that contain only user data.
- The `activation_date` field is now optional in any of the schemas, as described in the documentation.
- Fixed some minor file formatting issues.

whisller commented on json/schema/user_import.schema.json:
```diff
@@ -0,0 +1,191 @@
+{
+  "$id": "https://lifeworks.com/onboarding/json/v1/full_import.schema.json",
```
Shouldn't schema be `user_import.schema.json`?

whisller commented on json/schema/user_import.schema.json:
```diff
@@ -0,0 +1,191 @@
+{
```
Don't we require `company_id` as well? Without it we wouldn't know where to put user to : >

whisller replied:
Also I think we require `company_name` regardless of what type of import it is, at least based on https://github.com/workivate/onboarding-file-formats#properties

matepager replied:
This is meant for user-only uploads where all employees in the file belong to a single company and the company details are retrieved from another source, for example in the admin panel the company details will depend on which company is the user managing when they use the upload feature to manage employee accounts. I'll update the README to explain this.

matepager committed:
Update readme and fix schema ID/description

- Updated README to explain the function of the user import schemas.
- Fixed $id in user import JSON schema.
- Fixed description of the user import XSD.
